lms/lmsweb/views.py

```diff
@@ -208,6 +208,50 @@ def share():
     return fail(400, f'Unknown or unset act value "{act}".')
 
 
+@webapp.route('/notes/<int:user_id>', methods=['GET', 'POST'])
+@login_required
+def note(user_id: int):
+    if not current_user.role.is_manager:
```

Comment:
What if the user have public notes?

Reply:
This function is only for creating and deleting notes.
A user can't do any of that.
The fetching is directly in the user template file.